BackendTaskPilot/appsettings.json

```
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*"
```

Add CNB base url configuration
```suggestion
  "AllowedHosts": "*",
  "ExternalServices": {
    "CnbBaseUrl": "https://api.cnb.cz/"
  }
```

See [Configuration in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-8.0)  to use this config